src/dune/cram.mli

```diff
@@ -1,6 +1,6 @@
 open Import
 
-type test =
+type test = Cram_test.t =
```

Comment:
I think you can remove `Cram.test` altogether and move everything into `cram_tets`. The rest can remain as `cram_stanza`.

Reply:
Been changed.